### PR TITLE
[ZT] specify domains to include in split tunnels

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/connections/connect-devices/warp/exclude-traffic/split-tunnels.md [hunk 1]
```diff
@@ -67,6 +67,15 @@ Domain-based split tunneling works alongside DNS by dynamically excluding or inc
...
 {{</Aside>}}
 
+### Cloudflare Zero Trust domains
+
+Many Cloudflare Zero Trust services rely on traffic going through WARP. If you are using Split Tunnels in Include mode, you will need to manually add the following domains in order for these features to function.
+
+| Feature | Domains to include |
+| --------| ------------------ |
+| [Device posture checks](/cloudflare-one/identity/devices/)| `<your-team-name>.cloudflareaccess.com`, IdP used to authenticate into Access, and the application protected by Access |
+| [WARP session duration](/cloudflare-one/policies/filtering/enforce-sessions/) | `<your-team-name>.cloudflareaccess.com>` |
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/exclude-traffic/split-tunnels.md]: I don't believe the cloudflareaccess.com page is required for network session re-auth. However this is a good callout and we should specify something like "The Gateway HTTP or NETWORK resource with the session re-auth policy must be going through the tunnel for it to trigger".

Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/warp/exclude-traffic/split-tunnels.md]: The cloudflareacess.com requirement was mentioned in a CUSTESC ticket: https://jira.cfops.it/browse/PCX-3879

Comment (kkrum) [reply content/cloudflare-one/connections/connect-devices/warp/exclude-traffic/split-tunnels.md]: Huh, TIL, thanks for the link!

Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/exclude-traffic/split-tunnels.md]:
```suggestion
| [WARP session duration](/cloudflare-one/policies/filtering/enforce-sessions/) | `<your-team-name>.cloudflareaccess.com>` and the HTTP or Network resource with the session re-authentication policy |
```

**Review by kkrum — APPROVED** [review]

**Review by mmalden — APPROVED** [review]

**Pull request merged**